package-lock.json

```diff
@@ -57,7 +57,7 @@
     "@sinonjs/samsam": {
       "version": "3.0.2",
       "resolved": "https://registry.npmjs.org/@sinonjs/samsam/-/samsam-3.0.2.tgz",
-      "integrity": "sha512-m08g4CS3J6lwRQk1pj1EO+KEVWbrbXsmi9Pw0ySmrIbcVxVaedoFgLvFsV8wHLwh01EpROVz3KvVcD1Jmks9FQ==",
+      "integrity": "sha1-ME+zO9VYWgst+KTIAfy0f6hNjkM=",
```

Comment:
This stuff here shouldn't happen.

Which npm version are you using?

Reply:
I'm using `6.4.1`... Sorry about that. Using `6.5.0` gives me `sha512` hashes as expected. 

What do you think of the idea of adding `"npm": "^6.5.0" ` in `devDependencies`?

Reply:
I didn't add `npm` in `devDependencies` but used a newer npm version to update `package-lock.json` in commit 78f3dd8a583cca869617aaee7bc1c56e69713e87.

Putting `npm` in `devDependencies` would make sense to me. That way we would all use the exact same version by running `./node_modules/.bin/npm`. We can discuss that later :)

Reply:
Hmm. You're suggesting listing `npm` as a dependency of things to install with `npm`? That's sounds strange. I've never seen that before. Maybe it's a thing though.

Reply:
> You're suggesting listing `npm` as a dependency of things to install with `npm`?

Said that way it sounds kinda stupid indeed :laughing:. We could instead update the `CONTRIBUTING.md` to state what version of `npm` is required.

Reply:
We use the `engines` property in `package.json` for this in Dash:
https://github.com/plotly/dash-table/blob/03b59fd53cea0c61a42a3124bb126b24d29d51b3/package.json#L91-L94, https://docs.npmjs.com/files/package.json#engines

Reply:
@etpinard, see @chriddyp's comment above. That's exactly the feature I was looking for: don't let me change `package-lock.json` if my npm version is wrong! Maybe we should add this here and in plotly.js' repo.